src/VirtualClient/VirtualClient.Common/SSHClient/ISshClientProxy.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace VirtualClient.Common
+{
+    using System;
+    using Renci.SshNet;
```

Comment:
Do you know how popular this package is? Is there something from .NET?

Reply:
<img width="953" alt="image" src="https://github.com/microsoft/VirtualClient/assets/29044654/715cb328-b92c-4d97-aeee-92a2d687bbd2">

Reply:
77.8 million downloads in total for library.